### ISSUE TITLE
Bug: Wrong aria-label and tooltips on components

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-styles#79]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.

- [ ] Back button should be labeled as "Back" instead of "Previous". 
       - Action bar
       - Calendar ("Previous month", "Next month")
       - Date picker ("Previous month", "Next month")

- [ ] Description Yes/No is invalid. Instead, use "Delete" and "Cancel".
       - Model 

- [ ] If a user tabs to the objects, the user must know what element he has selected and how many other items are in the current group. (ie. "Option 1, item 1 of 4")
       - Combobox input
       - Contextual menu
       - Dropdown
       - Mega menu
       - Menu
       - Side navigation

- [ ] The back button should receive appropriate tooltips. 
       - Action bar
       - Calendar
       - Context menu
       - Date picker
       - Dropdown

**Comment by droshev** [comment]
most of the components have been refactored
